test/test_tensor.py

```diff
@@ -105,41 +105,6 @@ def test_dropout(self):
     expected = n * (1 - rate)
     np.testing.assert_allclose(non_zeros, expected, rtol=2e-3)
 
-  #@unittest.skipUnless(Device.DEFAULT == Device.CPU, "float64 not supported on GPU")
-  @unittest.skip("float64 support broken")
```

Comment:
If we remove this shouldn't we remove extra/gradcheck.py

Reply:
> If we remove this shouldn't we remove extra/gradcheck.py

Why?

Reply:
Can you fix this test instead of removing it?

Reply:
> Can you fix this test instead of removing it?

Sure, I'll see what I can do

Reply:
would also merge with this test skipped and not removed

Reply:
> would also merge with this test skipped and not removed

I think it's cleaner if we merge now. I'll open a new PR that fixes those tests instead